### PR TITLE
React Native/Ionic legacy docs

### PR DESCRIPTION
**Pull request by jordanranz**
*Description of changes:*
- Migrate legacy React Native docs
- Most file changes are directory changes (need to sync with Harry on this)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by harrysolovay** [comment]
@jordanranz let me know when you're ready for me to pull this into the updated URL branch (which, I'm about to push and draft)

File: docs/ui-legacy/api/fragments/react-native/connect.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+The API category provides React components for working with GraphQL data using the Amplify GraphQL client.
```
Comment (sammartinez) [inline docs/ui-legacy/api/fragments/react-native/connect.md:1]:
```suggestion
The API category provides React Native components for working with GraphQL data using the Amplify GraphQL client. 
```

Maybe?

File: docs/ui-legacy/api/fragments/react-native/connect.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    }
+} 
+
+export default App;
+
+```
+
+## Subscription
+
+Also, you can use the `subscription` and `onSubscriptionMsg` attributes to enable subscriptions:
```
Comment (sammartinez) [inline docs/ui-legacy/api/fragments/react-native/connect.md:46]:
```suggestion
You can use the `subscription` and `onSubscriptionMsg` attributes to enable subscriptions:
```

File: docs/ui-legacy/auth/fragments/ionic/authenticator.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
+AWS Amplifies provides UI components that you can use in your view templates.
```
Comment (sammartinez) [inline docs/ui-legacy/auth/fragments/ionic/authenticator.md:1]:
```suggestion
AWS Amplify provides UI components that you can use in your view templates. 
```

File: docs/ui-legacy/auth/fragments/ionic/authenticator.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
+AWS Amplify provides UI components that you can use in your view templates. 
+
+The Authenticator component creates an drop-in user authentication experience. Add the `amplify-authenticator` component to your `app.component.html` view:
```
Comment (brene) [inline docs/ui-legacy/auth/fragments/ionic/authenticator.md:3]: creates "a" drop-in user...

File: docs/ui-legacy/auth/fragments/ionic/authenticator.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+```html
+  <amplify-authenticator framework="Ionic"></amplify-authenticator>
+```
+
+### SignUp Configuration
+The SignUp component provides your users with the ability to sign up.  It is included as part of the ```authenticator``` component, but can also be used in isolation:
```
Comment (brene) [inline docs/ui-legacy/auth/fragments/ionic/authenticator.md:10]: shouldn't it be ```amplify-authenticator```

File: docs/ui-legacy/auth/fragments/ionic/authenticator.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+}
+```
+
+In `app.component.html`:
+```html
+<amplify-authenticator [signUpConfig]="signUpConfig"></amplify-authenticator>
+```
+
+### Sign up/in with email/phone number
+If the user pool is set to allow email addresses/phone numbers as the username, you can then change the UI components accordingly by using `usernameAttributes`.
```
Comment (brene) [inline docs/ui-legacy/auth/fragments/ionic/authenticator.md:81]: / => or

**Pull request merged**